src/components/ChatLog.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, handleLikeClick }) => {
+    return (
+        <div className="chat-log">
+            {entries.map((entry) => (
+                <ChatEntry
+                    {...entry}
+                    key={entry.id}
+                    handleLikeClick={handleLikeClick}
+                />
+            ))}
+        </div>
+    );
+};
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.number.isRequired,
+            sender: PropTypes.string.isRequired,
+            body: PropTypes.string.isRequired,
+            timeStamp: PropTypes.string.isRequired,
+            liked: PropTypes.bool.isRequired,
+        })
+    ).isRequired,
+        handleLikeClick: PropTypes.func,
```

Comment:
Nice job with the prop types here. Small nitpick, line 30 should be un-indented one level so that it is aligned to the same indentation of `entries` on line 21